docs/source/en/_toctree.yml

```diff
@@ -19,6 +19,8 @@
     title: Train a diffusion model
   - local: tutorials/using_peft_for_inference
     title: Inference with PEFT
+  - local: tutorials/interrupting_diffusion_process.md
```

Comment:
I think it'd be better to add this as a section in the existing [Pipeline callback](https://huggingface.co/docs/diffusers/main/en/using-diffusers/callback) guide. This is a very specific topic that is better documented as part of a larger topic (like callbacks in general).